### ISSUE TITLE
[MultiComboBox]: Accessibility: Checkboxes within the MultiComboBox should be of at least 24px as per guidelines

### ISSUE THREAD

**Issue by abhijeet181088** [SAP/ui5-webcomponents-react#5829]
### Describe the bug

Currently the size of checkboxes within MultiComboBox are following:
Normal: 22px (1.375rem)
Compact: 16px (1rem)

Accessibility guidelines for size of target elements:
**_"Targets have an area of at least a 24 CSS pixels diameter circle, including dedicated surrounding space, unless it is within a sentence or block of text."_**

> 

### Isolated Example

https://stackblitz.com/edit/github-pngbdn?file=src%2FApp.tsx

### Reproduction steps

1. Go to the isolated stackblitz example link provided.
2. Click on the first MultiComboBox (normal) and inspect the checkbox within it.
3. Size of checkbox is 22px (1.375rem)
4. Click on the second MultiComboBox (compact) and inspect the checkbox within it.
5. Size of checkbox is 16px (1rem)
...

### Expected Behaviour

Size of checkboxes should be at least 24px as per accessibility guidelines.

### UI5 Web Components for React Version

1.28.1

### UI5 Web Components Version

1.24.0

### Browser

Chrome

### Operating System

macOS

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
